ui/article-renderer/src/main/java/org/cru/godtools/article/activity/CategoriesActivity.java

```diff
@@ -0,0 +1,63 @@
+package org.cru.godtools.article.activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.MainThread;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.FragmentManager;
+
+import org.cru.godtools.article.R;
+import org.cru.godtools.article.fragment.CategoriesFragment;
+import org.cru.godtools.base.tool.activity.BaseSingleToolActivity;
+
+import java.util.Locale;
+
+public class CategoriesActivity extends BaseSingleToolActivity {
+    private static final String TAG_MAIN_FRAGMENT = "mainFragment";
+
+    public static void start(@NonNull final Context context, @NonNull final String toolCode,
+                             @NonNull final Locale language) {
```

Comment:
I never thought of creating a static method to start an activity it makes so much since.

Reply:
it's a really nice convenience method :) it ensures you are passing the correct data in the correct locations.